ui/components/app/user-preferenced-currency-display/user-preferenced-currency-display.component.js

```diff
@@ -24,9 +26,10 @@ export default function UserPreferencedCurrencyDisplay({
   showCurrencySuffix,
   ...restProps
 }) {
-  const currentNetwork = useSelector(getCurrentNetwork);
-  const nativeCurrency = useSelector(getNativeCurrency);
+  const currentNetwork = useSelector(getMultichainCurrentNetwork);
+  const nativeCurrency = useSelector(getMultichainNativeCurrency);
   const { currency, numberOfDecimals } = useUserPreferencedCurrency(type, {
+    currency: nativeCurrency, // Required for non-EVM support for now
```

Comment:
I don't see currency used in `useUserPreferencedCurrency`

Reply:
IIRC, I use it for the bitcoin account overview, I'll remove it from this PR!